sly.el

```diff
@@ -7077,10 +7077,10 @@ The result is unspecified if there isn't a symbol under the point."
   "Return the bounds of the symbol around point.
 The returned bounds are either nil or non-empty."
   (let ((bounds (bounds-of-thing-at-point 'sly-symbol)))
-    (if (and bounds
```

Comment:
This change is merely cosmetic if I understand correctly. Leave it out of this PR (if you want, submit it in another one)...

Reply:
Returned `if`.